pubsub/redis/redis.go

```diff
@@ -20,14 +20,27 @@ import (
 )
 
 const (
-	host              = "redisHost"
-	password          = "redisPassword"
-	consumerID        = "consumerID"
-	enableTLS         = "enableTLS"
-	processingTimeout = "processingTimeout"
-	redeliverInterval = "redeliverInterval"
-	queueDepth        = "queueDepth"
-	concurrency       = "concurrency"
+	host               = "redisHost"
+	password           = "redisPassword"
+	db                 = "redisDB"
+	maxRetries         = "maxRetries"
```

Comment:
My apologies, I didn't catch this before. In other components this field is called `backoffMaxRetries`, mainly because the [github.com/cenkalti/backoff](https://github.com/cenkalti/backoff) library is used.
Please rename this to `backoffMaxRetries`. It's just a consistency thing across components even though the Redis setting is differently named.